services/openvpn/service/manager.go

```diff
@@ -160,14 +160,22 @@ func (m *Manager) ProvideConfig(sessionConfig json.RawMessage, pingerPort func(i
 		}
 		if m.isBehindNAT() && m.portMappingFailed() {
 			c.Port = pp.Num()
-			pingerPort(pp.Num())
+			if m.serviceOptions.Port > 0 {
```

Comment:
This if looks fishy. If we provide a specific port, we default to 12234. Why is that?

Reply:
It is fishy :) Added TODO notice. Welcome to take :) It might be good first issue.

Reply:
Logic basically says that we need to have a separate port for pinger if static port is configured. We cannot run both pinger and service on the same port at the same time.